pkg/haprovider/haprovider.go

```diff
@@ -108,6 +120,7 @@ func (r *HAProvider) createService(
 		},
 		Spec: corev1.ServiceSpec{
 			Type: corev1.ServiceTypeLoadBalancer,
+			IPFamilies: []corev1.IPFamily{corev1.IPFamily(primaryIPFamily)},
```

Comment:
Let's add a TODO comment here to remind that when ako fully supports the dual stack load balancer type of service, we need to change it to add two IP families here.